src/signal.rs

```diff
@@ -1,18 +1,9 @@
+use crate::control::Control;
 use futures::Stream;
 
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
-pub enum SignalTo {
-    /// Signal to reload config.
-    Reload,
-    /// Signal to shutdown process.
-    Shutdown,
-    /// Shutdown process immediately.
-    Quit,
-}
-
 /// Signals from OS/user.
 #[cfg(unix)]
-pub fn signals() -> impl Stream<Item = SignalTo> {
```

Comment:
The name `Control` seems like a regression in clarity, at least to me. I don't really know what a "control" is in the abstract, while "signal" imples how we're using it. Is there something you didn't feel like `Signal` conveyed?

Reply:
No, `Signal` was perfectly fine IMO, it just felt specific to its current usage as "OS signals" -- whereas `Control` was a top-level controlling encompassing _both_ signals, and the new provider control messages.

It was just a new name, for a new type.

Happy to change this to whatever you'd rather it be called.